osis-core/src/main/java/com/scality/osis/service/impl/ScalityRedisRepository.java

```diff
@@ -0,0 +1,45 @@
+/**
+ *Copyright 2021 Scality, Inc.
+ *SPDX-License-Identifier: Apache License 2.0
+ */
+
+package com.scality.osis.service.impl;
+
+import com.scality.osis.service.IRedisRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.redis.core.StringRedisTemplate;
+import org.springframework.stereotype.Repository;
+
+import static com.scality.osis.utils.ScalityConstants.DEFAULT_REDIS_HASH_KEY;
+
+@Repository
+public class ScalityRedisRepository implements IRedisRepository {
+
+    @Autowired
+    StringRedisTemplate redisTemplate;
+
+    @Value("${osis.scality.redis.hashKey}")
+    String osisRedisHashKey = DEFAULT_REDIS_HASH_KEY;
```

Comment:
It could be useful to add a "namespace" prefix to the key. It would help avoid conflicts with other services, and add a natural place to extend in the future. Something like `DEFAULT_REDIS_PREFIX:DEFAULT_REDIS_HASH_KEY` with `DEFAULT_REDIS_PREFIX` being `osi` or something else service related.

Reply:
fixed in 54ec8339065c3a9c01b4c5ef626dfc9c4a9fa054